boards/opentitan/src/alarm_test.rs

```diff
@@ -0,0 +1,31 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![allow(unused_imports,dead_code)]
+
+use capsules::test::alarm::TestAlarm;
+use kernel::hil::time::Alarm;
+use ibex::timer;
+use kernel::static_init;
+
+pub unsafe fn run_alarm() {
```

Comment:
I don't see any mention of this in the PR description, why add an alarm test?

Reply:
It will likely not be in the in the final PR, but it's here so it's easy to test the alarm.